README.md

```diff
@@ -198,15 +198,17 @@ The framework names in the directories should be lowercased and without special
 
 ### Linking between sections and guides
 
-When linking between sections in the same guide, simply us a relative link to the appropriate section name.  This will preserve whatever framework the user currently has selected.
+**Always** have a trailing slash at the end of your guides link.  (Example: `/guides/NAME-OF-GUIDE/-/NAME-OF-SECTION/`)
 
-When linking between different guides, use `-` in place of the framework name - this will default the framework to the first option in the StackSelector for that guide.  Example: `/guides/NAME-OF-GUIDE/-/NAME-OF-SECTION`
+When linking between sections in the same guide, simply us a relative link to the appropriate section name, which will be one directory back. (Example: `../OTHER-SECTION/`)  This will preserve whatever framework the user currently has selected.
 
-To link to another guide you MUST link to a section of that guide - incomplete guide URLs will fallback to the main Guide Overview page.
+When linking between different guides, use `-` in place of the framework name - this will default the framework to the first option in the StackSelector for that guide.  Example: `/guides/NAME-OF-GUIDE/-/NAME-OF-SECTION/`
+
+To link to another guide you can, but do not need to, link to a specific section.  If you link to just the guide, it will redirect the user to the first section using the default (first) framework option in the StackSelector.
 
 Within the content of a guide the `<NextSectionLink>` component is available to link to different sections:
 
 * `<NextSectionLink/>` - Provides a "button" link to the next section
 * `<NextSectionLink>Some Example Text</NextSectionLink>` - Provides the "button" with different text
-* `<NextSectionLink name="next-steps"/>` - Provides the "button" to link to the named section of the guide (doesn't have be "next")
+* `<NextSectionLink name="next-steps"/>` - Provides the "button" to link to the named section of the guide (doesn't have be the "next" section)
```

Comment:
`<NextSectionLink name="next-steps"/>` - Provides the "button" to link to the named section of the guide (doesn't have to be.......